analytics/aggregate.go

```diff
@@ -549,6 +549,11 @@ func AggregateData(data []interface{}, trackAllPaths bool, ignoreTagPrefixList [
 			continue
 		}
 
+		// We don't want to aggregate Graph Data with REST data - there is a different type for that.
+		if thisV.IsGraphRecord() {
```

Comment:
@pvormste I think the best approach for aggregated records is to start receiving a bool flag and `if ignoreGraphQL && thisV.IsGraphRecord()` here. 
Why? Because there's another pump - `sql aggregate` that uses this function and it's going to generate different analytics between "normal" SQL and aggregated SQL.

Reply:
Yeah so for now, i changed it so only data trying to be aggregated by the mongo aggregate pump is filtered.